### PR TITLE
Add swift-numerics

### PR DESCRIPTION
**Pull request by clackary**
### Pull Request Description

Add https://github.com/apple/swift-numerics

### Acceptance Criteria
- [x] be an *Xcode* or *swift package manager* project
- [x] support building on either Linux or macOS
- [x] target Linux, macOS, or iOS/tvOS/watchOS device
- [x] be contained in a publicly accessible git repository
- [x] maintain a project branch that builds against Swift 4.0 and passes any unit tests
- [x] have maintainers who will commit to resolve issues in a timely manner
- [x] be compatible with the latest GM/Beta versions of *Xcode* and *swiftpm*
- [x] add value not already included in the suite
- [x] be licensed with one of the following permissive licenses:
	* Apache License, version 2.0
- [x] pass `./project_precommit_check` script run
```
PASS: swift-numerics, 5.2, ae364a, Swift Package
========================================
Action Summary:
     Passed: 1
     Failed: 0
    XFailed: 0
    UPassed: 0
      Total: 1
========================================
Repository Summary:
      Total: 1
========================================
Result: PASS
========================================
--- swift-numerics checked successfully against Swift 5.2 ---
```

### CODE REVIEW

**Comment by clackary** [comment]
@swift-ci test

File: projects.json [hunk 1]
```diff
@@ -2963,6 +2963,31 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/apple/swift-numerics",
+    "path": "swift-numerics",
+    "branch": "main",
+    "maintainer": "stephentyrone@gmail.com",
```
Comment (shahmishal) [inline projects.json:2971]: Is there a reason this is personal email vs apple email?

Comment (clackary) [reply projects.json:2971]: Just using the one listed on GitHub account.

@stephentyrone Do you have a preference for which email is listed here?

**Comment by shahmishal** [comment]
Can you also add tag "tags": "swiftpm"?

**Comment by clackary** [comment]
@swift-ci test

**Comment by clackary** [comment]
@swift-ci test

**Comment by clackary** [comment]
@shahmishal This should be ready to merge once it is approved.

**Pull request merged**